src/app.js

```diff
@@ -10,37 +10,14 @@ import { INTERNAL_SERVER_ERROR } from 'http-codes';
 import { v4 as uuidv4 } from 'uuid';
 import crypto from 'crypto';
 
+import {} from './processHandler';
 import { hsesAuth } from './middleware/authMiddleware';
 import { retrieveUserDetails } from './services/currentUser';
 import cookieSession from './middleware/sessionMiddleware';
 
 import { logger, auditLogger, requestLogger } from './logger';
 import runCronJobs from './lib/cron';
 
-process.on('_fatalException', (err) => {
-  logger.error('Fatal exception', err);
-  process.exit(1);
-});
-
-process.on('uncaughtException', (err) => {
-  logger.error('Uncaught exception', err);
-  process.exit(1);
-});
-
-process.on('unhandledRejection', (reason, promise) => {
```

Comment:
this code was preventing the CI from crashing on the maxretriesperrequest error. Is it implemented correctly below? We seem to be having the crashes it was preventing again (the backend crashes, the remaining tests time out waiting for a response)